utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -54,7 +54,8 @@ class UrlImageParser private constructor(
     val imageUrl = "$gcsPrefix/$gcsResourceName/$imagePath"
     val proxyDrawable = ProxyDrawable()
     // TODO(#1039): Introduce custom type OppiaImage for rendering Bitmap and Svg.
-    val isSvg = machineLocale.run { imageUrl.endsWithIgnoreCase("svg") }
+    val isSvg = machineLocale.run { imageUrl.endsWithIgnoreCase("svg") } ||
+      machineLocale.run { imageUrl.endsWithIgnoreCase("svgz") }
```

Comment:
```suggestion
    val isSvg = machineLocale.run {
      imageUrl.endsWithIgnoreCase("svg") ||imageUrl.endsWithIgnoreCase("svgz")
    }
```

Can be simplified a bit.

Reply:
Done.